### ISSUE TITLE
Class: RequestError - Invalid auction currency

### ISSUE THREAD

**Issue by gerpaick** [timotheus/ebaysdk-python#212]
Hi, 
i am running this problem: I  want to list an item on ebay in uk. i used token i registered on UK version of ebay. all api call i changed from .com to co.uk. 
in sample trading code i change cuurency to GBP but what it accept is USD only.
i am running everything in sandbox. 
here is code:
`myitem = {
            "Item": {
                "Title": item.name,
                "Description": "<![CDATA[{}]]>".format(item.description),
                "PrimaryCategory": {"CategoryID": "377"},
                "StartPrice": str(round(item.price.amount, 2)),
                "CategoryMappingAllowed": "true",
                "Country": "GB",
                "ConditionID": "3000",
                "Currency": "GBP",
                "DispatchTimeMax": "3",
                "ListingDuration": "Days_30",
                "ListingType": "FixedPriceItem",
                "PaymentMethods": "PayPal",
                "PayPalEmailAddress": "aa.zz@xxx.com",
                "PictureDetails": {"PictureURL": "http://i1.sandbox.ebayimg.com/03/i/00/30/07/20_1.JPG?set_id=8800005007"},
                "PostalCode": "DH129BY",
                "Quantity": str(round(item.qty, 0)),
                "ReturnPolicy": {
                    "ReturnsAcceptedOption": "ReturnsAccepted",
                    "RefundOption": "MoneyBack",
                    "ReturnsWithinOption": "Days_30",
                    "Description": "If you are not satisfied, return the goods for refund.",
                    "ShippingCostPaidByOption": "Buyer"
                },
                "ShippingDetails": {
                    "ShippingType": "Flat",
                    "ShippingServiceOptions": {
                        "ShippingServicePriority": "1",
                        "ShippingService": "USPSMedia",
                        "ShippingServiceCost": "2.5"
                    }
                },
                "Site": 'UK'
            }
        `}``

and the answer:
`'AddFixedPriceItem: Class: RequestError, Severity: Error, Code: 95, Invalid auction currency. The auction currency specified does not match the auction currency for the selected site.'
{'Errors': {'ErrorCode': '95', 'ShortMessage': 'Invalid auction currency.', 'SeverityCode': 'Error', 'ErrorClassification': 'RequestError', 'LongMessage': 'The auction currency specified does not match the auction currency for the selected site.'}, 'Build': 'E1031_UNI_API5_18532394_R1', 'Ack': 'Failure', 'Timestamp': '2017-09-25T18:48:34.353Z', 'Version': '1031'}
`

any idea how to fix it?`

**Comment by Bobspadger** [comment]
I did have a similar issue, from memory there were a couple of issues

1. The sandbox I was using was not set up for my currency (vague memory - had to ask on ebay dev forums???)
2. I needed to put the auction site in my config.yml file 


api.sandbox.ebay.com:
    compatability: 987
    appid: MY_APP_ID
    siteid: 3
    certid: MY_CERT_ID
    devid:      MY_DEV_ID


As I mentioned before , I'm at home and not got all my info to hand. hope these help.
